### PR TITLE
More maintenance

### PR DESCRIPTION
**Pull request by YashdalfTheGray**
* added `hubot-hello` so hubot will say hi
* cleaned up the `package.json` file and added a `package-lock.json`
* added `eslint` for linting the javascript and a `npm run linter` command to lint the entire project
* added an `.eslintignore` file so that `node_modules` and `bin` aren't linted
* changed `editorconfig` indent for json and yaml to 2 spaces.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -10,3 +10,9 @@ insert_final_newline = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.yml]
+indent_size = 2
+
+[*.json]
+indent_size = 2
```
Comment (mikestephens) [inline .editorconfig]: Mah man

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "extends": "airbnb-base",
+    "plugins": [
+        "import"
+    ],
+    "rules": {
+        "brace-style": ["error", "stroustrup"],
+        "class-methods-use-this": ["off"],
+        "comma-dangle": ["error", "never"],
```
Comment (mikestephens) [inline .eslintrc.json]: Mah man

Comment (YashdalfTheGray) [reply .eslintrc.json]: 🤜 🤛

File: .editorconfig (hunk 1 above)
Comment (YashdalfTheGray) [reply .editorconfig]: 🤜 🤛

**Pull request merged**